lib/datadog/tracing/contrib/status_range_matcher.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Datadog
+  module Tracing
+    module Contrib
+      # Useful checking whether the defined range covers status code
+      class StatusRangeMatcher
+        def initialize(ranges)
+          @ranges = Array(ranges)
+        end
+
+        def include?(status)
+          @ranges.any? do |e|
+            case e
+            when Range
+              e.include? status
```

Comment:
How about using === for both integer and range? I think it works for both.

Reply:
Compared to current code I feel like it kind of obscures the reading though: personally I prefer reading the current version which is immediately obvious, whereas I had to check that  `2 === 2` and `(2..4) === 2` both worked.